org/org.yaml

```diff
@@ -800,20 +794,21 @@ orgs:
       chains.collaborators:
         description: The chains collaborators
         maintainers:
+        - lcarva
+        - priyawadhwa
+        - wlynch
+        members:
```

Comment:
Group maintainers can administrate the group - historically I configured only GB members in the maintainers as in fact the groups are only configured via this org file via a service account.
Group members is what should be used in all groups, whether it's `*.collaborators`, `*.maintainers` or *.admin`

Reply:
of course group maintainers are also members, so this works, but it gives users more privileges than they need

Reply:
Moved everyone to members!